tests/libbpfgo.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+#
+# This test is executed by github workflows inside the action runners
+#
```

Comment:
Maybe a little text about what this test does would be good

Reply:
Done.

```
# This test attempts to compile tracee using upstream libbpfgo.
# It's run by github workflows inside action runners, just as
# it can be run locally. In both cases it must be triggered by
# 'make test-upstream-libbpfgo'.
```